cmd/manager/main.go

```diff
@@ -50,9 +65,18 @@ func main() {
 	log.Info(fmt.Sprintf("KUDO Version: %s", fmt.Sprintf("%#v", version.Get())))
 
 	// create new controller-runtime manager
-	log.Info("setting up manager")
+	syncPeriod, err := parseSyncPeriod()
+	if err != nil {
+		log.Error(err, "unable to parse manager sync period variable, run manager with defaults")
+	} else if syncPeriod != nil {
```

Comment:
Let's not chain these `if`-clauses as they're checking different things
```suggestion
	}
	if syncPeriod != nil {
```